Content.Shared/Kitchen/SharedKitchenSpikeComponent.cs

```diff
@@ -23,15 +24,13 @@ public abstract class SharedKitchenSpikeComponent : Component, IDragDropOn
 
         bool IDragDropOn.CanDragDropOn(DragDropEventArgs eventArgs)
         {
-            if (eventArgs.User == eventArgs.Dragged ||
-                !eventArgs.Dragged.TryGetComponent<IMobStateComponent>(out var state) ||
-                (eventArgs.User.TryGetComponent(out SharedCombatModeComponent? combatMode) && !combatMode.IsInCombatMode))
+            if (!eventArgs.Dragged.HasComponent<SharedButcherableComponent>())
             {
                 return false;
             }
 
             // TODO: Once we get silicons need to check organic
-            return !state.IsDead();
+            return true;
```

Comment:
this reverts the kitchenspike requiring the following things:
- thing is dead
- user is in combatmode
- user cant spike themselves (i dunno honestly, should probably be possible?)

which were implemented to make it a less efficient killing machine.

why?

Reply:
For all of those, I think they should be moved to error messages in the TrySpike method.

- I removed the IsDead check because it has messages for that after the SpikeDelay which is 12 seconds.
- Combat mode was confusing a lot of players because there was no error message, so I removed it as a requirement.
- The self spike gives a message to the player, but I guess I can change it to only work on suicide.

I don't think it's that much of a killing machine since it has a 12 second DoAfter which cancels if the victim moves, or the user gets hurt, stunned, switches hands, or moves.

Reply:
forgot that the doafter got added, however i cant find an isdead message after the spikedelay. otherwise its fine

Reply:
I removed the IsDead check because I don't think it's needed with the 12 second delay. I can add it back if the spike should only work on dead.